src/lib.rs

```diff
@@ -27,7 +27,7 @@
 
 pub extern crate bitcoin;
 #[macro_use]
-pub extern crate bitcoin_hashes;
+pub extern crate bitcoin_hashes as do_not_use_me_just_imported_for_the_macros;
```

Comment:
Lol yeah I don't think it needs to be a `pub use`, does it? Then it doesn't matter what name it has.

Reply:
loll yeah good call